Maple2.Tools/Collision/PointPrism.cs

```diff
@@ -26,7 +26,7 @@ public bool Intersects(IPrism prism) {
     }
 
     private readonly struct Point(float x, float y) : IPolygon {
-        public bool Contains(in Vector2 point) {
+        public bool Contains(in Vector2 point, float epsilon = 1e-5f) {
             const float tolerance = 0.0000001f;
             return Math.Abs(x - point.X) < tolerance && Math.Abs(y - point.Y) < tolerance;
```

Comment:
`epsilon` not used, and `tolerance` is the same, so might as well replace uses of `tolerance` with `epsilon`.